src/scss/ui/_tag.scss

```diff
@@ -0,0 +1,68 @@
+.tag-container {
+    display: flex;
+    flex-wrap: wrap;
+
+    .tag {
+        background-color: $tag-bg-color;
+        height: 23px;
+        border: 1px solid gray;
+        padding: 0 .2rem;
```

Comment:
all paddings, margins, etc should be multiple of 4 and written in rems. There should be `0 .25rem`